src/tutorials/abilities_advanced.md

```diff
@@ -804,7 +795,7 @@ The main thing to change for our ammo supply is to change the properties of the
 
         ```lua
         function OnBeginOverlap(whichTrigger, other)
-            if other:IsA("Player") then
+            if other and other:IsA("Player") then
```

Comment:
Go back. In many of these functions there's an assumption that an object will exist. People can be paranoid in their own codes, but we don't have to be paranoid here.